File: app/src/main/java/ar/com/wolox/android/example/ui/login/LoginActivity.java
package ar.com.wolox.android.example.ui.login;

import android.os.Bundle;
import android.os.PersistableBundle;

import ar.com.wolox.android.R;
import ar.com.wolox.wolmo.core.activity.WolmoActivity;

/**
 *
 */
public class LoginActivity extends WolmoActivity {

    @Override
    public void onCreate(Bundle savedInstanceState, PersistableBundle persistentState) {
        super.onCreate(savedInstanceState, persistentState);
        init();
    }

    @Override
    protected int layout() {
        return R.layout.activity_base;
    }

    @Override
    protected void init() {
        replaceFragment(R.id.vActivityBaseContent, new LoginFragment());
    }
}

File: app/src/main/java/ar/com/wolox/android/example/ui/login/LoginActivity.kt
package ar.com.wolox.android.example.ui.login

import ar.com.wolox.android.R
import ar.com.wolox.wolmo.core.activity.WolmoActivity

class LoginActivity : WolmoActivity() {

    override fun init() {
        replaceFragment(R.id.vActivityBaseContent, LoginFragment())
    }

    override fun layout(): Int = R.layout.activity_base
}